README.md

```diff
@@ -64,6 +64,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [CloudFactory](https://www.cloudfactory.com/) | Bhaisepati, Lalitpur | On-Demand Workforce for Data Work
 - [Cloudy Fox Technology Pvt. Ltd.](https://www.cloudyfox.io) | Anamnagar, Kathmandu | Software Development, Support, Maintenance & Security
 - [Clusus Information Solutions Pvt. Ltd.](http://www.clusus.com) | Jhamsikhel, Lalitpur | Leading provider of banking and payment solutions focused bestowing excellence and distinction to the Banking and Financial industry
+-[codekarkhana](https://codekarkhana.com/) | Sankhaml, Kathmandu | Software Development, SEO, Hosting reseller
```

Comment:
@g0dgamerz Thank you for the contribution.

However, I won't be able to merge this change because Code Karkhana doesn't seem to have a valid website at this point. It is just a template with no information about the company.